pkg/graveler/graveler.go

```diff
@@ -29,6 +29,8 @@ const (
 
 	SettingsRelativeKey = "%s/settings/%s.json" // where the settings are saved relative to the storage namespace
 
+	BrnachUpdateMaxInterval = 5
```

Comment:
We need the unit as part of the interval
```suggestion
	BrnachUpdateMaxInterval = 5 * time.Second
```

Reply:
Done